core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -112,14 +147,12 @@ class MultiCloudWorkspaceService(userInfo: UserInfo,
   }
 
   private def createWorkspace(workspaceRequest: MultiCloudWorkspaceRequest, parentSpan: Span): Future[Workspace] = {
-    val azureConfig = multiCloudWorkspaceConfig.azureConfig.getOrElse(throw new RawlsException("Azure config not present"))
     val wsmConfig = multiCloudWorkspaceConfig.workspaceManager.getOrElse(throw new RawlsException("WSM app config not present"))
 
-    // TODO these will come from the spend profile service in the future
-    val spendProfileId = azureConfig.spendProfileId
-    val azureTenantId = azureConfig.azureTenantId
-    val azureSubscriptionId = azureConfig.azureSubscriptionId
-    val azureResourceGroupId = azureConfig.azureResourceGroupId
+    val spendProfileId = workspaceRequest.billingProfileId
```

Comment:
Hopefully spendProfileId is the same thing as the billingProfileId.

Reply:
It should be.